ksql-engine/src/test/java/io/confluent/ksql/function/udaf/min/MinUdafTest.java

```diff
@@ -0,0 +1,155 @@
+/*
+ * Copyright 2019 Confluent Inc.
+ *
+ * Licensed under the Confluent Community License (the "License"; you may not use
+ * this file except in compliance with the License. You may obtain a copy of the
+ * License at
+ *
+ * http://www.confluent.io/confluent-community-license
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OF ANY KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+
+package io.confluent.ksql.function.udaf.min;
+
+import static org.hamcrest.CoreMatchers.equalTo;
+import static org.junit.Assert.assertThat;
+
+import io.confluent.ksql.function.udaf.Udaf;
+import org.junit.Test;
+
+public class MinUdafTest {
+
+  @Test
+  public void shouldMinLongs() {
+    final Udaf<Long, Long, Long> udaf = MinUdaf.minLong();
+    final long[] values = new long[]{3L, 5L, 8L, 2L, 3L, 4L, 5L};
+    long currentMin = Long.MAX_VALUE;
+    for (final long i: values) {
+      currentMin = udaf.aggregate(i, currentMin);
+    }
+    assertThat(2L, equalTo(currentMin));
```

Comment:
The parameters to `assertThat` are the wrong way around. The first param should be the _actual_ value and the second the _expected_ value. Otherwise the error messages come out all wrong. e.g. `expected 4 but get 2`.

```suggestion
    assertThat(currentMin, equalTo(2L));
```

Same for other tests in this file and other files.